modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -653,7 +652,7 @@ export default class Ecdsa {
             s: bigIntToBufferBE(proof.s, 384).toString('hex'),
             s1: bigIntToBufferBE(proof.s1, 96).toString('hex'),
             s2: bigIntToBufferBE(proof.s2, 480).toString('hex'),
-            t1: bigIntToBufferBE(proof.t1, 416).toString('hex'),
```

Comment:
Seems this is a breaking change. Can you kindly mention all breaking changes in the commit footer

Reply:
This is not a breaking change as this would only affect the HSM and the HSM code is still in review